components/mongodb/actions/update-a-document/update-a-document.mjs

```diff
@@ -0,0 +1,58 @@
+import mongodbApp from "../../mongodb.app.mjs";
+
+export default {
+  key: "mongodb-update-a-document",
+  name: "Update a Document",
+  description: "Updates a single document by ID. [See the docs here](https://docs.mongodb.com/manual/reference/method/db.collection.findOneAndUpdate/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mongodbApp,
+    collection: {
+      propDefinition: [
+        mongodbApp,
+        "collection",
+      ],
+    },
```

Comment:
The description for this is "Collection where the new document will be inserted". It should be something like "Collection containing the document to be updated."